packages/reports/src/Containers/profit-table.tsx

```diff
@@ -109,7 +109,8 @@ const ProfitTable = observer(({ component_icon }: TProfitTable) => {
     >;
 
     const mobileRowRenderer: React.ComponentProps<typeof DataList>['rowRenderer'] = ({ row, is_footer }) => {
-        const duration_type = /^(MULTUP|MULTDOWN)/.test(row?.shortcode) ? '' : row?.duration_type;
+        const duration_type = row?.duration_type;
+        const formatted_duration_type = duration_type ? duration_type[0].toUpperCase() + duration_type.slice(1) : '';
```

Comment:
`formatted_duration_type` will be translated, correct?

Reply:
It comes translated from API I suppose